upgrading/manual_upgrades.adoc

```diff
@@ -507,80 +507,38 @@ updated.
 
-After a cluster upgrade, the default roles
+During a cluster upgrade, the default roles
 xref:../architecture/additional_concepts/authorization.adoc#roles[default
-cluster roles] are automatically updated. To check if all defaults are set as
-recommended for your environment, run:
-
-----
-# oc adm policy reconcile-cluster-roles
-----
+cluster roles] are automatically updated. 
 
 [WARNING]
 ====
-If you have customized default cluster roles and want to ensure a role reconciliation
-does not modify those customized roles, annotate them with `openshift.io/reconcile-protect`
-set to `true` when using the old Openshift policy format. When using the new RBAC
-roles, use `rbac.authorization.kubernetes.io/autoupdate` set to `false` instead.
-In doing so, you are responsible for manually updating those roles with any new
-or required permissions during upgrades.
+If you customized default cluster roles and want to ensure a role reconciliation
+does not modify those customized roles, set the
+`rbac.authorization.kubernetes.io/autoupdate` parameter to `false`.
+
+You must manually update the roles that contain this setting to include any new
+or required permissions after upgrading.
 ====
 
-This command outputs a list of roles that are out of date and their new proposed
-values. For example:
-
-----
-# oc adm policy reconcile-cluster-roles
-apiVersion: v1
-items:
-- apiVersion: v1
-  kind: ClusterRole
-  metadata:
-    creationTimestamp: null
-    name: admin
-  rules:
-  - attributeRestrictions: null
-    resources:
-    - builds/custom
-...
+. Generate a default bootstrap policy template file:
++
 ----
-
+$ oc adm create-bootstrap-policy-file --filename=policy.json
+----
++
 [NOTE]
 ====
-Your output will vary based on the OpenShift version and any local
-customizations you have made. Review the proposed policy carefully.
+The contents of the file vary based on the OpenShift version, but the file 
+contains only the default policies.
 ====
 
-You can either modify this output to re-apply any local policy changes you have
-made, or you can automatically apply the new policy using the following process:
+. Update the *_policy.json_* file to include any cluster role customizations.
 
-. Reconcile the cluster roles:
-+
-----
-# oc adm policy reconcile-cluster-roles \
-    --additive-only=true \
-    --confirm
-----
-
-. Reconcile the cluster role bindings:
-+
-----
-# oc adm policy reconcile-cluster-role-bindings \
-    --exclude-groups=system:authenticated \
-    --exclude-groups=system:authenticated:oauth \
-    --exclude-groups=system:unauthenticated \
-    --exclude-users=system:anonymous \
-    --additive-only=true \
-    --confirm
-----
-+
-Also run:
+. Use the policy file to automatically reconcile the roles and role bindings:
 +
 ----
-# oc adm policy reconcile-cluster-role-bindings \
-    system:build-strategy-jenkinspipeline \
-    --confirm \
-    -o name
+$ oc process --local -f policy.json | oc auth reconcile -f policy.json
```

Comment:
The whole command here should be just the part after the pipe;  `oc auth reconcile -f policy.json`